feature-odyssey/src/main/kotlin/com/hedvig/android/odyssey/search/GetClaimEntryPointsUseCase.kt

```diff
@@ -1,12 +1,11 @@
 package com.hedvig.android.odyssey.search
 
+import arrow.core.Either
+import com.hedvig.android.core.common.ErrorMessage
 import com.hedvig.android.odyssey.model.SearchableClaim
 
 internal interface GetClaimEntryPointsUseCase {
-  suspend operator fun invoke(): CommonClaimsResult
+  suspend operator fun invoke(): Either<ErrorMessage, CommonClaimsResult>
```

Comment:
Hmm I know it was this way before too, but when using the `operator fun invoke` instead of just `fun invoke` we are bumping into this bug https://youtrack.jetbrains.com/issue/KTIJ-1050/Go-to-implementation-fails-on-an-override-invoke-function-call-in-both-operator-and-function-forms where trying to do Cmd + B (Or Cmd + click) on the function it does not find its usages when calling this function.
It's an annoying IDE bug which has been there since forever, but imo just getting rid of the `operator` is our best bet, since we can still call it by doing `useCase.invoke()` instead of `useCase()`, and it's consistent with the rest of the code base.